cl/users/views.py

```diff
@@ -418,7 +418,7 @@ def delete_account(request: AuthenticatedHttpRequest) -> HttpResponse:
     )
 
 
-def delete_profile_done(request: HttpRequest) -> HttpResponse:
+async def delete_profile_done(request: HttpRequest) -> HttpResponse:
     return TemplateResponse(request, "profile/deleted.html", {"private": True})
```

Comment:
I'm curious what the point is of making this kind of micro-view async, since there's very little IO going on here. Is it just to help with the loading of the template from disk? Does django even do that async?

Reply:
Making a view natively async means django will be able to handle it more efficiently by not having to use a threadpool. It's more efficient this way essentially.

Reply:
Ah, I see. Because if it's not native async, Django has to fake it. We're talking milliseconds, right?

Reply:
Yeah, pretty much, django detects if a view function is async or not to decide if it needs to run the view function in a threadpool.